### PR TITLE
Announce Lucky 0.18.0

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: src/posts/lucky_0.18.cr [hunk 1]
```diff
@@ -0,0 +1,156 @@
+class Lucky018Release < BasePost
+  title "Lucky 0.18 is out now."
+  slug "lucky-0_18-release"
+
+  def published_at : Time
+    Time.utc(year: 2019, month: 10, day: 7)
```
Comment (paulcsmith) [inline src/posts/lucky_0.18.cr:6]: Release on Monday

File: src/posts/lucky_0.18.cr [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    We've also introduced a way to easily report errors. Lucky generates
+    an `Errors::Show` class with a new `report` method that is empty. You can
+    fill in the method to report it however you want.
+
+    For example, we could use the [Raven shard](https://github.com/sija/raven.cr)
+    to send an error report to [Sentry](https://sentry.io/):
+
+    ```crystal
+    # src/actions/errors/show.cr
+    def report(error : Exception)
```
Comment (jwoertink) [inline src/posts/lucky_0.18.cr:107]:
```suggestion
    private def report(error : Exception) : Nil
```

When we generate this, it's a private method that returns nil. I think that `Raven.capture` does return nil, so the example is probably fine.

**Review by jwoertink — COMMENTED** [review]
Awesome! Thanks for taking care of this. Just one small comment, but otherwise it's ready to go!

**Review by paulcsmith — COMMENTED** [review]
Thanks for the review and code suggestion!

File: src/posts/lucky_0.18.cr [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      Raven.capture(error)
+    end
+    ```
+
+    Read more about [error reporting](#{Guides::HttpAndRouting::ErrorHandling.path(anchor: Guides::HttpAndRouting::ErrorHandling::ANCHOR_REPORTING)}).
+
+    ### Keep your database and schema in sync
+
+    We've added a new feature called Schema Enforcer that runs when your server
+    is started and in development or test mode. **If you have a missing column, table, or incorrect type,** Lucky will let you know.
```
Comment (edwardloveall) [inline src/posts/lucky_0.18.cr]: Odd or missing line break here, but not a big deal.

**Review by edwardloveall — APPROVED** [review]
Looks great!

**Comment by paulcsmith** [comment]
Thanks for the review!

**Pull request merged**